examples/dht_node.rs

```diff
@@ -79,7 +80,10 @@ fn main() {
     // Create a channel for server to communicate with network
     let (tx, rx) = mpsc::unbounded::<(DhtPacket, SocketAddr)>();
 
+    let net_crypto = NetCrypto::new(tx.clone(), server_pk, server_sk.clone());
+
     let mut server_obj = Server::new(tx, server_pk, server_sk);
+    server_obj.set_net_crypto(net_crypto);
```

Comment:
When won't the Server have NetCrypto? is it possible?

Reply:
It might be possible when we start only bootstrap node, without the messenger. But actually it can be created with `NetCrypto` as well.

Reply:
Got it